recipes/libharu/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "libharu" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libharu/libharu/archive/RELEASE_{{ version.replace('.', '_') }}.tar.gz
+  sha256: 8f9e68cc5d5f7d53d1bc61a1ed876add1faf4f91070dbc360d8b259f46d9a4d2
+  patches:
+    - 0001-skip-installing-if-directory.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+  host:
+    - zlib
+    - libpng
+    - boost
```

Comment:
Do you need `Boost.Python`? Otherwise you should depend on `boost-cpp`.

Reply:
I'm not sure why @hobu added `boost` at all.  I have removed it and things build fine.